docs/operation/Recover_From_Etcd_Permanent_Quorum_Loss.md

```diff
@@ -0,0 +1,83 @@
+## Quorum loss in ETCD Cluster
+[Quorum loss](https://etcd.io/docs/v3.3/op-guide/recovery/) means when majority of ETCD pods(Greater than or equal to n/2 + 1) are down simultaneously for some reason. 
```

Comment:
Please link the latest documentation, or atleast to etcd `v3.4`, since druid uses etcd `v3.4` version at the moment.